src/index.html

```diff
@@ -13,10 +13,14 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file (`main.scss`). Browsers do not understand SCSS directly; it needs to be compiled into CSS first. Ensure that you compile your SCSS file into a CSS file and link to the resulting CSS file instead.